pegasus/sites.v3/code.org/views/plc_csd_nav.haml

```diff
@@ -1,15 +1,15 @@
 #about_nav
   %span.about_list
-    -current_url = '/educate/professional-learning/cs-discoveries'
+    -target_url = '/educate/professional-learning/cs-discoveries'
     -{'Middle School Courses'=>'/educate/curriculum/middle-school',
       'About the Professional Learning Program'=>'#about',
       'How does it work?'=>'#components',
       'What are the commitments?'=>'#commitments',
       'Who can participate?'=>'#participate',
       'Where is it offered?'=>'#locations',
       'How to apply?'=>'#apply'}.each_pair do |label,url|
```

Comment:
Actually that's not right.  It officially seems to be a fragment identifier: https://en.wikipedia.org/wiki/Fragment_identifier